### ISSUE TITLE
Teamplay - statistic

### ISSUE THREAD

**Issue by Micr0Bit** [OpenRA/OpenRA#11452]
i thought we could track teamplay-actions (or support-actions) and reward players for that at the end of a game. ideas for that : 

* placing defense-stuctures (pillbox, coils, turrets, flame-towers) inside a team-members buildradius (not overlapped by your own buildradius)
* killing enemy-units from within a team-members buildradius (not overlapped by your buildradius)
* repair bridges
* sending supply-trucks to a team-member
* unloading harvester on team-members refinery
* reloading team-members units
* transporting team-members unit x-amount of cells
* repairing team-members building with an engineer or repair-button

Faction-Specific
* heal/repair team-members units with medic/mechanic
* use spyplane/paratroops

**Comment by HenrytheSlav** [comment]
Big warning - easily abusable with force fire. We hardly have any point score system in place. I think we should first establish some.

**Comment by Micr0Bit** [comment]
you can also get a lot of kills on purpose ... if you play in a certain way ... so you can abuse that too ... or getting deaths , apm ...

Big Warning , thats not the point here ... 

im just asking to reward teamplay-actions with points ... if the players abuse that for whatever reason it doesnt matter ...

**Comment by HenrytheSlav** [comment]
Fair point, but it's true that we lack any sensible scoring system. So we need to establish that first (possibly including these sugestions ofc).

**Comment by pchote** [comment]
#10358 has been sitting there for a long time as a first step towards this.

**Comment by abcdefg30** [comment]
From what I heard @GraionDilach wanted to take that over. Not sure if he still has time and will though.
